scripts/decrypt_secrets

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
```

Comment:
the use of `set -e` caused a problem when one of the functions made a call that returned a non-zero return code, causing the script to terminate prematurely (see the commit I made with the script fix).  A better way might be to not rely on `set -e` and to instead explicitly check calls when needed and exit on failure so the impacted function that was fixed doesn't have to toggle `set -e` off and on... unless there's another fix for that issue that I'm not aware of.

Reply:
`set -e` makes sense here